src/main/java/dev/tycho/stonks/command/base/SubCommand.java

```diff
@@ -11,39 +11,32 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public abstract class CommandSub {
+public abstract class SubCommand {
```

Comment:
This class should remain named as CommandSub  to reduce diff and to better autocomplete in IDEs